### ISSUE TITLE
Portable rock doesn't always show at top of stack

### ISSUE THREAD

**Issue by mrkubax10** [SuperTux/supertux#1781]
**SuperTux version: latest master when submitting this issue**
**System information:** *any supported OS*

##### Expected behavior
When releasing action key when standing in front of rock stack rock should appear on top of stack.
##### Actual behavior
Rock sometimes appear on top of stack, sometimes shows above player failling at them.
##### Steps to reproduce actual behavior
Grab rock, and release action key when in front of rock stack.

**Comment by Guih48** [comment]
> ##### Expected behavior
> When releasing action key when standing in front of rock stack rock should appear on top of stack.

I don't agree the expected behaviour is if you can't place the holdable object next to you and you release the action key, you still hold rhe holdable object until you can't place (or throw) it normally (this case often apperars if you holding an object next to a vall, and also facing to the wall.
